platform/iphone/app_delegate.h

```diff
@@ -33,6 +33,7 @@
 // Old accelerometer approach deprecated since IOS 7.0
 // Include coremotion for accelerometer, gyroscope and magnetometer access, available since IOS 4.0 but some functionality won't work for anything before IOS 5.0
 #import <CoreMotion/CoreMotion.h>
+#import <QuartzCore/QuartzCore.h>
```

Comment:
Given this file isn't used in the .h, maybe it can be included in the .mm (I don't know iOS though, so don't trust me on that)